### PR TITLE
refactor: update app item data

### LINKED ISSUE #293: Update App Item props

### PR DESCRIPTION
**Pull request by pyphilia**
close #293

### CODE REVIEW

**Review by swouf — APPROVED** [review]
Ça me semble ok.

File: src/utils/membership.js [hunk 1]
```diff
@@ -18,6 +12,30 @@ export const isItemUpdateAllowedForUser = ({ memberships, memberId }) =>
     ),
   );
 
+export const getBestPermissionForMemberFromMemberships = ({
```
Comment (juancarlosfarah) [inline src/utils/membership.js:15]: Maybe just add a description of what is `best`? From what I see here quickly, it's by length?

File: src/components/main/ItemScreen.js [hunk 2]
```diff
@@ -53,10 +54,12 @@ const ItemScreen = () => {
     return <ErrorAlert />;
   }
 
-  const enableEdition = isItemUpdateAllowedForUser({
+  const itemMembership = getBestPermissionForMemberFromMemberships({
```
Comment (juancarlosfarah) [inline src/components/main/ItemScreen.js:57]: Would be good to explain what `best` is?

**Review by juancarlosfarah — CHANGES_REQUESTED** [review]
@pyphilia, looking good! Just think that we should define in the function and where it's used what `best` means in this context.

**Pull request merged**